src/var/isFunction.js

```diff
@@ -7,7 +7,8 @@ define( function() {
 		// In some browsers, typeof returns "function" for HTML <object> elements
 		// (i.e., `typeof document.createElement( "object" ) === "function"`).
 		// We don't want to classify *any* DOM node as a function.
-		return typeof obj === "function" && typeof obj.nodeType !== "number";
+		return typeof obj === "function" && typeof obj.nodeType !== "number" &&
+			[ "[object HTMLCollection]", "[object NodeList]" ].indexOf( obj.toString() ) === -1;
```

Comment:
Please add a support comment mentioning affected environments & their versions. Examples are all the comments starting with `// Support:`, you can find many examples in the code base.